packages/plugin-dev/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@theia/plugin-dev",
+  "version": "0.6.0",
+  "description": "Theia - Plugin Development Extension",
+  "main": "lib/common/index.js",
+  "typings": "lib/common/index.d.ts",
+  "dependencies": {
+    "@theia/core": "^0.6.0",
```

Comment:
it would be good to declare only dependencies which are actually used in, seems to be copy-paste from plugin-ext

Reply:
Done